### ISSUE TITLE
Runtime Exception in Flutter 1.8.1-pre.44

### ISSUE THREAD

**Issue by aloklearning** [aloisdeniel/flutter_geocoder#23]
The new version of flutter throws an error in the package. Can we get the updated package, cos it is very good, just some code is casing issue. 

```
file:///Users/alok/flutter/.pub-cache/hosted/pub.dartlang.org/geocoder-0.1.2/lib/services/d
istant_google.dart:38:56: Error: The argument type 'Utf8Decoder' can't be assigned to the
parameter type 'StreamTransformer<Uint8List, dynamic>'.
 - 'Utf8Decoder' is from 'dart:convert'.
 - 'StreamTransformer' is from 'dart:async'.
 - 'Uint8List' is from 'dart:typed_data'.
Try changing the type of the parameter, or casting the argument to
'StreamTransformer<Uint8List, dynamic>'.
    final responseBody = await response.transform(utf8.decoder).join();
```

Let me know whe you fix it. Thanks

**Comment by stuartmorgan-g** [comment]
See https://github.com/aloisdeniel/flutter_geocoder/pull/22

**Comment by aloisdeniel** [comment]
Thanks I’ll update signatures.

**Comment by aloisdeniel** [comment]
It should be fixed [0.2.1 version](https://pub.dev/packages/geocoder/versions/0.2.1).

Thanks for reporting!
